R/pkg/R/mllib.R

```diff
@@ -299,6 +307,92 @@ setMethod("summary", signature(object = "NaiveBayesModel"),
             return(list(apriori = apriori, tables = tables))
           })
 
+# Returns posterior probabilities from a Latent Dirichlet Allocation model produced by spark.lda()
```

Comment:
I'll add `\link{spark.lda}` in `write.ml`. 

As for `predict`, LDA uses the name `spark.posterior`. So I'll leave `predict` unchanged.